### ISSUE TITLE
Custom label not working?

### ISSUE THREAD

**Issue by zhurin** [linuxmint/cinnamon-spices-desklets#919]
```
 * Desklet version/Build date: Version 1.2 (2023-08-07 19:35:12)
 * Cinnamon version: 5.8.4
 * Distribution: Mint 21.2
 * Graphics hardware *and* driver used: Intel HD Graphics 530, driver: i915
 * Snippet or attachment of ~/.cinnamon/glass.log: file not found
 * Snippet or attachment of ~/.xsession-errors: see section, "Other information"

 ```

**Notify author of desklet**
@schorschii

**Issue**
I don't see the custom label and there's no screenshot on https://cinnamon-spices.linuxmint.com/desklets/view/49 to reference where it should appear or how it should look.

**Steps to reproduce**

1. Open Configure dialog for Host Check
2. Verify **Hide decorations** is disabled
3. Enable **Use a custom desklet label**
4. Enter label text in **Custom desklet label** text box
5. Close Configure dialog

**Expected behaviour**
Custom label text appears somewhere visible on the desklet, in a tooltip, on the notification popup, etc.

**Observed behaviour**
No custom label found. It still looks the same as the example screenshot with status on the first line, host name/IP on the second, and background color based on the current status.

**Other information**
I currently have two desklets, which have their settings stored in 6.json and 7.json in ~/.config/cinnamon/spices/hostcheck@schorschii/ and they match the settings and custom label as configured in the UI. However, when changing a setting in the UI, a seemingly bogus (since the file is updated appropriately) file-not-found error appears in the .xsession-errors log:

```
Traceback (most recent call last):
  File "/usr/share/cinnamon/cinnamon-settings/bin/JsonSettingsWidgets.py", line 136, in check_settings
    self.settings = self.get_settings()
  File "/usr/share/cinnamon/cinnamon-settings/bin/JsonSettingsWidgets.py", line 151, in get_settings
    file = open(self.filepath)
FileNotFoundError: [Errno 2] No such file or directory: '/home/###/.config/cinnamon/spices/hostcheck@schorschii/7.json'
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 153, in apport_excepthook
    with os.fdopen(os.open(pr_filename,
FileNotFoundError: [Errno 2] No such file or directory: '/var/crash/_usr_share_cinnamon_cinnamon-settings_xlet-settings.py.1000.crash'

Original exception was:
Traceback (most recent call last):
  File "/usr/share/cinnamon/cinnamon-settings/bin/JsonSettingsWidgets.py", line 136, in check_settings
    self.settings = self.get_settings()
  File "/usr/share/cinnamon/cinnamon-settings/bin/JsonSettingsWidgets.py", line 151, in get_settings
    file = open(self.filepath)
FileNotFoundError: [Errno 2] No such file or directory: '/home/###/.config/cinnamon/spices/hostcheck@schorschii/7.json'
```

**Comment by schorschii** [comment]
This is the custom label:
![custom-label](https://github.com/linuxmint/cinnamon-spices-desklets/assets/17480795/651521b2-cb9c-4bb9-81c8-d7c9e8e1992a)
As stated in the desklet settings, it is only visible if decorations are enabled (a system-wide desklets setting, not specific to my desklet).
![image](https://github.com/linuxmint/cinnamon-spices-desklets/assets/17480795/b9b6dab8-0ef9-4cbd-a0e0-e01fa09b2668)
I'm currently not able to reproduce the error you mentioned. It also looks more like a general Cinnamon Desklets issue rather than a specific problem of my desklet.

**Comment by zhurin** [comment]
Thanks for the clarification and confirmation screenshot. In the Host Check settings, it says, "only visible if decorations are enabled," which sound like they're referring to the Host Check setting for decorations, so I overlooked the fact that the global desklet settings had decorations disabled.

I now see the custom label as illustrated by your screenshot. Yay!

Regarding the file-not-found errors, it's OK if you're unable to reproduce them since I've confirmed they aren't affecting the Host Check operation. I just included the info in case it was useful.

Thanks for the help and for the cool desklet!
